### DISCUSSION TITLE
Welcome to CrimeStop-Analytics Discussions!

### DISCUSSION THREAD

**Discussion by GarretTomlin** [Crime-Analytica/CrimeStop-Analytics#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GarretTomlin** [comment]
@100010We  Welcome our new member he will be the UI and Backend dev of the team

@strique18

**Comment by 100010We** [comment]
Welcome, looking forward to collaborating and learning a few stuff from you @strique18

**Comment by GarretTomlin** [comment]
@100010We @strique18 are You guys available next week Sunday for a meeting?

**Reply by 100010We** [reply]
Yhh

**Comment by strique18** [comment]
Hey! Guys I am Tyrique Hayles aka strique, I will be working on UI and backend. I am a 2nd year com science student currently on leave. I hold a certificate in android development using java in android studio. I want to be a game developer, I have put that dream on hold for now to join new projects and improved my skill. I am very happy to be a part of this project!

**Reply by GarretTomlin** [reply]
> Hey! Guys I am Tyrique Hayles aka strique, I will be working on UI and backend. I am a 2nd year com science student currently on leave. I hold a certificate in android development using java in android studio. I want to be a game developer, I have put that dream on hold for now to join new projects and improved my skill. I am very happy to be a part of this project!

Happy to have you on the team

**Comment by GarretTomlin** [comment]
![news](https://github.com/Crime-Analytica/CrimeStop-Analytics/assets/65048014/a34ae5b7-a2fc-410c-9f7b-4d4fcea904f7)

**Reply by GarretTomlin** [reply]
@strique18 @100010We  btw I haven't told you guys what's in it for you, basically when the company does take off stock options and a competitive salary plus job security those are just the few

**Comment by GarretTomlin** [comment]
@strique18  @100010We  welcome @JavianTheGreek  he will be one of the core people on the team if he chooses to join this project,he will be joining us as the lead UI/UX designer

**Reply by strique18** [reply]
Welcome bro @JavianTheGreek

**Reply by 100010We** [reply]
Welcome looking forward to our future interactions

**Comment by JavianTheGreek** [comment]
hi there everyone I'm Javian Cummings, a third-year computer science student that is passionate about building something fresh and engaging that will have an influence on the world. I'll be focusing on Ux/UI and I am thrilled to be a part of this team.

**Reply by GarretTomlin** [reply]
Happy to have you on broad

**Reply by 100010We** [reply]
welcome

**Comment by GarretTomlin** [comment]
@strique18 @100010We  @100010We Hey team please Join the slack workspace this will be our primary means of communications

https://join.slack.com/t/crimeanalytica/shared_invite/zt-1xgnvxjeb-N7sLqRPaDzwviYN8NcRE7A
